### PR TITLE
Change how to make the task_id

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: pallets/automation-time/src/tests.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use crate::{mock::*, Error, OverlflowTasks, Task, Tasks};
+use crate::{mock::*, Error, OverlflowTasks, Task, TaskHashInput, Tasks};
```
Comment (irsal) [inline pallets/automation-time/src/tests.rs:1]: what about a test for colliding `provided_id`'s

File: pallets/automation-time/src/lib.rs [hunk 2]
```diff
@@ -131,6 +150,8 @@ pub mod pallet {
 		PastTime,
 		/// The message cannot be empty.
 		EmptyMessage,
+		/// The provided_id cannot be empty
+		EmptyProvidedId,
```
Comment (irsal) [inline pallets/automation-time/src/lib.rs:154]: dupe provided id would just be a dupe task right?

Comment (rhuttman) [reply pallets/automation-time/src/lib.rs:154]: correct

File: pallets/automation-time/src/tests.rs (hunk 1 above)
Comment (rhuttman) [reply pallets/automation-time/src/tests.rs:1]: that is in there: https://github.com/OAK-Foundation/OAK-blockchain/pull/58/files#diff-8aa2921b8ffd2c8a5901003639e03bb443a2032d62c687606ea6a9f7bbcaf99cR114

**Review by irsal — APPROVED** [review]
🦅

**Pull request merged**